### ISSUE TITLE
Hide/unhide "Your Name" field in help desk widget or contact form

### ISSUE THREAD

**Issue by libcce** [stagecc/bdc3-website-public#47]
The "Your Name" field appears in the help desk widget but not in the contact form. A BDC3 member has asked for us to maintain consistency and add the "Your Name" field to the contact form. Unfortunately, the Freshdesk admin interface doesn't allow us to turn this field off for the widget or add it to the contact form. We have submitted this as an issue to Freshdesk but in the meantime... Freshdesk has offered the following two links where fields can be hidden/unhidden:

https://support.freshdesk.com/support/solutions/articles/50000000032-prepopulating-hiding-or-disa…

https://developers.freshdesk.com/widget-api/#disable-fields

If possible, we ask for the "Your Name" field on the contact form to be unhidden. If this is not possible, then hiding the field on the help desk widget would be our second choice.

This issue is low priority.

**Comment by mbwatson** [comment]
i'm not familiar with the freshdesk utilities offered, and i don't just want to make changes, but can a name text field be added with this interface: https://bdcatalyst.freshdesk.com/a/admin/ticket_fields ?

**Comment by libcce** [comment]
Hi @mbwatson I tried adding another "Your Name" field and it created two fields of the same name on the widget, so not what we want, and incorrectly added the field to the contact form. For the moment, can we label this as a "Freshdesk" issue. I have submitted a bug report to them, but I'm going to bring it up to them again. Stay tuned. Again, I see this as low priority at the moment but let's log it.

**Comment by libcce** [comment]
Hi @mbwatson, after checking in with Freshdesk looks like our only option to ensure that we have consistency across the widget and the contact form is to disable the "name" field from the widget. 

The solution is above https://github.com/stagecc/bdc3-website-public/issues/47#issue-621374210 where you include:

`FreshworksWidget('disable', 'ticketForm', ['subject', 'description', 'priority', 'status', 'group_id', 'product_id', 'type', 'custom_fields.cf_order_id']);`

Except we only want to disable the 'name' field. 

We have a meeting with Freshdesk regarding 508/accessibility compliance so best to wait on this issue until after that meeting.

**Comment by mbwatson** [comment]
in staging. will merge into master oday.

![Screenshot from 2020-05-26 16-36-16](https://user-images.githubusercontent.com/6019870/82948049-0efefe80-9f6f-11ea-82ac-a4927a922510.png)
